src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/TeleporterSystem.java

```diff
@@ -45,7 +33,11 @@
     @In
     EntityManager entityManager;
 
-    private Random random = new Random();
+    private final Random random = new Random();
+
+    private final int maxTeamSizeDifference = 3;
```

Comment:
This is a _configuration_ that might as well come from a component, and thus could be modified/set by a prefab.

Reply:
Regarding this I am actually looking into creating a GameplayConfiguration file which makes use of AutoConfig and will keep track of the maxTeamSizeDifference and any future configurations required with respect to LaS. As most of the configurations make use of it , the required UI is automatically created and the UI will be generated only when LaS is active this should be useful. This is something I didn't really look into or test out properly so it might not work the way I expected it to.  So I am planning on taking a look at that too before I decide if I make use of a Config file or convert it into a component.

Reply:
This actually requires a lot of additional setup and was not as concise as I hoped it would be.  So I mostly will be creating a component for this too and making use of commands to set the maxTeamSizeDifference.